usermanagementv1/user_management_v1_examples_test.go

```diff
@@ -212,6 +211,57 @@ var _ = Describe(`UserManagementV1 Examples Tests`, func() {
 			fmt.Println(string(b))
 			// end-list_users
 		})
+		It(`ListUsers request search example`, func() {
+			fmt.Println("\nListUsers() result:")
+			// begin-list_users search
```

Comment:
Within the generated examples code, you cannot simply add new sections like this, because there can be at most one example code block for each operation.  The "// begin" and "// end" comments are markers for the begin and end of the code fragment to be used for the operation whose id is specified in those marker comments.
So there can only be one section with:
```
       // begin-list_users
       ...
       // end-list_users
```
This restriction is in place because we have a separate tool that can be used to extract these code fragments (one per operation) and compile them in a .json file that can be used when building the online API reference docs.

So, bottom line is that you can have only 1 example per operation and you need to leave the begin/end marker comments as they are.  If you want to incorporate the new parameters in the existing list_users example, feel free to do that.

Reply:
I see, Thank you for the suggestion. Addressed.